academic-services/knowledge-exploration-service/reference-makes-command-line-tool.md

```diff
@@ -171,37 +194,74 @@ The input entities file Url prefix.
 
 The output url prefix for writing the built index.
 
-### Optional Parameters
+### BuildIndex Optional Parameters
 
 `--IndexPartitionCount`
 
 The number of index partitions to create. A index job can finish quicker when there are more partitions; however, the more partitions, the less accurate the interpret results will be. Maximize build performance by creating 1 partition per worker.
 
-`--IntersectionMinCount`
+`--IntersectionCountThresholdForPreCompute`
 
-The minimum intersections between indexed attribute for which the indexer should generate pre-calculated results. The higher the count, the more process will be required at run time. The lower the count, the larger the generated index will be.
+The attribute value intersections threshold for pre-computing look up tables. Use this value to tune index build time performance, run time performance, and index size. The higher the value, the smaller index size and slower run-time performance will be. The lower the value, the larger index size and faster run-time performance will be.
 
 `--MaxStringLength`
 
-The maximum string length for string type attributes.
-
-`--RemoveEmptyValues`
-
-Whether to remove empty attribute values to reduce index size.
+The maximum string length for all entity attributes. All strings over the maximum string length will be truncated
 
 `--WorkerCount`
 
 The number of virtual machines(workers) used to build the index. Warning, assigning a number larger than IndexPartitionCount won't result in performance gain.
 
-`--WorkerMachineSku`
+`--WorkerSku`
+
+The virtual machine(worker) sku. Use [List available VM Sizes](https://docs.microsoft.com/rest/api/compute/virtualmachines/listavailablesizes) to get the available options.
```

Comment:
>[List available VM Sizes](https://docs.microsoft.com/rest/api/compute/virtualmachines/listavailablesizes) [](start = 37, length = 105)

This seems like a really inconvenient way to get this information (having to make an API call)